### ISSUE TITLE
error

### ISSUE THREAD

**Issue by muggs88** [UnamSanctam/UnamBinder#15]
I binded two files. I binded a random file and your recent silent miner. I keep getting the same error. The first file pops up fine, but the second one I keep getting this error.
 Windows cannot find'C:\Users\Owner\AppData\Roaming\miner'. Make sure you typed the name correctly, and then try again.

Thanks.

**Comment by UnamSanctam** [comment]
Ah yeah there might be a bug with the starting string (not being encased in quotes), someone else reported it as well.

**Comment by muggs88** [comment]
Is there anything I can do about it?

**Comment by UnamSanctam** [comment]
If you compile it yourself then maybe, I think the issue is here:
https://github.com/UnamSanctam/UnamBinder/blob/647fca788bda11b316a2a9218e7f3f1144569366/UnamBinder/Resources/Program.c#L63
It should maybe be `"%s \"%s\""` there.

**Comment by muggs88** [comment]
so replace the "%s %s" with "%s \"%s\""

**Comment by muggs88** [comment]
my bad "%s \"%s\""

**Comment by muggs88** [comment]
The \ wont come up for some reason. I will give it a go.

**Comment by UnamSanctam** [comment]
Yes that might work. It auto formats the \ away when typing it since it's an escape character.

**Comment by muggs88** [comment]
Ok thanks. I will let you know how it goes.

**Comment by muggs88** [comment]
What do you use to compile?

**Comment by UnamSanctam** [comment]
Visual Studio 2019.

**Comment by muggs88** [comment]
Do you know any compilers that dont take up as much space as visual studios?

**Comment by muggs88** [comment]
Also, to compile it is all I do get the .sln and put it in visual studios then build?
Thanks btw.

**Comment by UnamSanctam** [comment]
Don't really know of any others that work well, but yeah just open the .sln file with Visual Studio and then build it.

**Comment by muggs88** [comment]
Mabye I messed something up but all I get is 2 command prompts that pop up.

**Comment by muggs88** [comment]
can you mabye give it a go if you have time. Im going to keep trying though.

**Comment by sweetpeps** [comment]
I'm also having this issue.

**Comment by sweetpeps** [comment]
> 

It actually just comes up with a CLI for each program you binded.
